kit/tracing/tracing.go

```diff
@@ -83,6 +83,9 @@ func StartSpanFromContext(ctx context.Context) (opentracing.Span, context.Contex
 	}
 	fn := runtime.FuncForPC(pcs[0])
 	name := fn.Name()
+	if lastPeriod := strings.LastIndexByte(name, '.'); lastPeriod > 0 {
+		name = name[lastPeriod+1:]
+	}
```

Comment:
Should we include the package name too (e.g. `LastIndexByte()` on `/` instead)? I'm not sure how much room is in the UI.

Reply:
@benbjohnson great question. Here are the specific pain points. @stuartcarnie and I talked about tradeoffs yesterday.

- The first screenshot demonstrates a user experience with no flexibility for string length unless we want to modify the Jaeger UI. Some function names, alone, will overflow the selector.
- The second screenshot demonstrates how spans are contextualized in a trace. Click a span and you get the filename and line number in a log. Because of these extra sources of context, I'm just taking the function name as "operation" for now.

![Screen Shot 2019-03-13 at 9 25 12 AM](https://user-images.githubusercontent.com/622263/54296564-66839b00-4572-11e9-9e04-d18f4e4b6f41.png)

![Screen Shot 2019-03-13 at 9 24 11 AM](https://user-images.githubusercontent.com/622263/54296567-6a172200-4572-11e9-9a7f-55ab6e43427a.png)

Reply:
I changed my mind. I think this screenshot demonstrates that there is too much ambiguity in just the function name. I'll follow up with another commit, going to slice from the last `/` rather than the last `.`. It would be cool to clean up the string a bit, but that would cost another allocation to build the string, rather than slicing into an existing string.

![Screen Shot 2019-03-13 at 9 41 47 AM](https://user-images.githubusercontent.com/622263/54297533-566cbb00-4574-11e9-8b4e-1cf848bf581d.png)